man/man4/zfs.4

```diff
@@ -2177,9 +2177,39 @@ for each I/O submitter.
 When unset, requests are handled asynchronously by a thread pool.
 The number of requests which can be handled concurrently is controlled by
 .Sy zvol_threads .
-.
-.It Sy zvol_threads Ns = Ns Sy 32 Pq uint
-Max number of threads which can handle zvol I/O requests concurrently.
+.Sy zvol_request_sync
+is ignored when running on a kernel that supports block multiqueue (
+.Li blk-mq
+).
```

Comment:
FWIW, non-critical, but this can probably look better as
```
.Pq Li blk-mq .
```

Reply:
actually scratch that, this version will get laid as "block multiqueue ( `blk-mq` )."; if you want to go for a literal (). then
```
.No ( Li blk-mq ) .
```
works (and renders right, as "block multiqueue (`blk-mq`)."), but Pq just works better here

Reply:
I eneded up going with `.Pq Li blk-mq .` since the other one gave me a stylecheck error:
```
mandoc: ./man/man4/zfs.4:2182:2: WARNING: skipping empty macro: No
```